feathr_project/feathr/spark_provider/_databricks_submission.py

```diff
@@ -274,4 +275,26 @@ def download_result(self, result_path: str, local_folder: str):
             raise RuntimeError('Currently only paths starting with dbfs is supported for downloading results from a databricks cluster. The path should start with \"dbfs:\" .')
 
         DbfsApi(self.api_client).cp(recursive=True, overwrite=True, src=result_path, dst=local_folder)
+        
```

Comment:
`upload_or_get_cloud_path` shall we reuse this method?

Reply:
this one doesn't support upload from cloud dir (will skip it for some reason)